openpype/hosts/photoshop/plugins/publish/collect_review.py

```diff
@@ -2,18 +2,26 @@
 
 import pyblish.api
 
+from openpype.lib import get_subset_name
+
 
 class CollectReview(pyblish.api.ContextPlugin):
     """Gather the active document as review instance."""
 
     label = "Review"
-    order = pyblish.api.CollectorOrder
+    order = pyblish.api.CollectorOrder + 0.1
     hosts = ["photoshop"]
 
     def process(self, context):
         family = "review"
-        task = os.getenv("AVALON_TASK", None)
-        subset = family + task.capitalize()
+        subset = get_subset_name(
```

Comment:
When you have access to asset document then use `get_subset_name_with_asset_doc` which does not query from mongo at all.